modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -118,6 +120,14 @@ public CouchbaseContainer(String containerName) {
         return Sets.newHashSet(getMappedPort(REST));
     }
 
+    @Override
+    protected void configure() {
+        if (clusterPassword.length() < REQUIRED_DEFAULT_PASSWORD_LENGTH) {
+            logger().warn("The provided cluster admin password length is less then the default password policy length. " +
+                "Cluster start will fail if configured password policy is offended.");
```

Comment:
As a native speaker, `offended` isn't a term I've seen used around password policies. Maybe this could be:

```suggestion
                "Cluster start will fail if configured password requirements are not met.");
```